apps/namadillo/src/integrations/Keplr.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  Window as KeplrWindow,
+  OfflineAminoSigner,
+  OfflineDirectSigner,
+} from "@keplr-wallet/types";
+import { ChainRegistryEntry } from "types";
+import { WalletInterface } from "./types";
+import { basicConvertToKeplrChain } from "./utils";
+
+type Signer = OfflineAminoSigner & OfflineDirectSigner;
+const keplr = (window as KeplrWindow).keplr!;
+
+export class KeplrWalletManager implements WalletInterface {
```

Comment:
One other thing I just realised when rebasing my PR to this one: if it all possible, I would really like not to abstract here with an interface until we're forced to by supporting other wallets. The reason is that we have to do some things that are specific to Keplr (in the case I'm thinking of, set and unset a Keplr option before and after submitting a transfer). If we use an interface for this, we'll need to add before/after hooks and it will get quite complicated. We had something like this before with the previous `Integration` interface, and ended up adding dummy methods in places where we needed to do things specific to one extension but not another. I think the future when we support a second extension would be a better time to add abstractions, since at that time we will know the exact requirements (and also that time may never come depending on requirements). What do you think?

Reply:
If we keep using a wallet abstraction, we could at least only require implementations for things that all wallets contain (where the interfaces can easily match), everything else could be added on to the specific wallet. Otherwise, it might be easier to just have 2 different types of wallets (eventually 3 with Metamask). Just a thought, I'm fine either way, so long as we can avoid those dummy implementations

Reply:
I'll keep this abstraction because I think it will be easier to remove it in the future than add other wallets if we don't have this implemented. But I renamed the interface to `WalletConnector`, to be something more specific.